### PR TITLE
Add new env for AML Data Labeling 

### PR DESCRIPTION
**Pull request by xiyao-ms**

### CODE REVIEW

File: data-labeling/environments/data-labeling/context/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,25 @@
+FROM mcr.microsoft.com/azureml/openmpi4.1.0-ubuntu20.04:{{latest-image-tag}}
+
+WORKDIR /
+
+ENV CONDA_PREFIX=/azureml-envs/labeling-base-ubuntu20.04-py39-cpu
```
Comment (elesel) [inline data-labeling/environments/data-labeling/context/Dockerfile:5]: Consider a simpler directory name:
```suggestion
ENV CONDA_PREFIX=/azureml-envs/labeling-base
```

File: data-labeling/environments/data-labeling/environment.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  os: linux
+  context:
+    dir: context
+    dockerfile: Dockerfile
+    template_files:
+    - Dockerfile
+    - conda_dependencies.yaml
+  publish:
+    location: mcr
+    visibility: unlisted
```
Comment (elesel) [inline data-labeling/environments/data-labeling/environment.yaml:12]: This is required for all new environments:
```suggestion
    visibility: public
```

File: data-labeling/environments/data-labeling/context/conda_dependencies.yaml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+- pip:
+  - azureml-core=={{latest-pypi-version}}
+  - azureml-contrib-dataset=={{latest-pypi-version}}
+  - azureml-dataset-runtime[fuse,pandas]
+  - azureml-defaults=={{latest-pypi-version}}
+  - azure-storage-blob==12.14.1
+  - Pillow=={{latest-pypi-version}}
+  - pycairo==1.20.1
+  - pydicom=={{latest-pypi-version}}
+  - python-rle==0.0.3
```
Comment (elesel) [inline data-labeling/environments/data-labeling/context/conda_dependencies.yaml:17]: Newline at end of file:
```suggestion
  - python-rle==0.0.3

```

File: data-labeling/environments/data-labeling/environment.yaml [hunk 4]
```diff
@@ -0,0 +1,12 @@
+image:
+  name: azureml/curated/data-labeling
```
Comment (elesel) [inline data-labeling/environments/data-labeling/environment.yaml:2]: Have you created a PR in the mcr repo for this image?

**Review by elesel — CHANGES_REQUESTED** [review]

File: data-labeling/environments/data-labeling/environment.yaml (hunk 4 above)
Comment (xiyao-ms) [reply data-labeling/environments/data-labeling/environment.yaml:2]: Created a PR in the mcr repo https://github.com/microsoft/mcr/pull/2529

Comment (xiyao-ms) [reply data-labeling/environments/data-labeling/environment.yaml:2]: Just merged the pr

**Review by elesel — APPROVED** [review]

**Pull request merged**